schemas/filetype.yml

```diff
@@ -74,3 +77,10 @@ classes:
                     Any keywords, phrases or classification codes that are relevant
                     to the file type, e.g., particular scientific domains of applicability,
                     or experimental techniques.
+    Container:
+        tree_root: true
+        attributes:
+            file_types:
+                multivalued: true
+                inlined_as_list: true
+                range: FileType
```

Comment:
What is this for?

Reply:
Something I was playing with from the [LinkML docs](https://linkml.io/linkml/intro/tutorial02.html)that is meant to streamline validation (otherwise `linkml-validate` needs to have the target type specified). I've reverted this for now as it doesn't seem to do what I want (and the pydantic schema for it isn't particularly good)